### DISCUSSION TITLE
Provides a simple list of results for the "pymobiledevice3 apps list" command.

### DISCUSSION THREAD

**Discussion by WeissRu** [doronz88/pymobiledevice3#522] [Ideas]
The "apps list" command produces a lot of output. It seems that a lot of information contained in the info.plist of the application will be displayed together. 
Can you provide a command that displays only simple information like name, bundle_id, version (one line per app), and then another command query the details based on the bundle_id?

**Comment by doronz88** [comment]
The command output is JSON so users can use `jq` to manipulate the data however they want.
For example, consider the following:
```shell
pymobiledevice3 apps list --system --no-color | jq '.[].CFBundleIdentifier'
```
